Saint-Nec-Engine/CommonLib/GameObject.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <vector>
+#include "Component.h"
+class GameObject
+{
+	std::vector<int> position;
```

Comment:
J'ai utilisé des vectors, tu préfères des tableaux pour forcer 3 exactement ?

Reply:
Maybe create a union struct or class to define vec3 from an array so we can call individual components (x, y, z)
There is already a vec 3 in OpenGL lib but I don't think it is a good idea to add opengl to non opengl code